api/src/components/publication/controller.ts

```diff
@@ -235,8 +236,22 @@ export const updateStatus = async (
         }
 
         // TODO, eventually a service in LIVE can be HIDDEN and a service HIDDEN can become LIVE
-        if (publication?.currentStatus !== 'DRAFT') {
-            return response.json(404, { message: 'A status of a publication that is not in DRAFT cannot be changed.' });
+        if (publication?.currentStatus === 'LIVE') {
+            return response.json(403, {
+                message: 'A status of a publication that is not in DRAFT or LOCKED cannot be changed.'
+            });
+        }
+
+        if (publication?.currentStatus === 'LOCKED') {
+            // Update status to 'DRAFT'
```

Comment:
This is not updating the status just to DRAFT, corresponding author can also update status to LIVE bypassing co-authors approvals...

We need to check the status provided by the user input also and if they are allowed to change the status from one to another

DRAFT -> LIVE
DRAFT -> LOCKED -> LIVE
DRAFT -> LOCKED -> DRAFT
LIVE -> nowhere...